test/freenet/client/filter/CSSParserTest.java

```diff
@@ -985,6 +986,8 @@ public class CSSParserTest {
 		propertyTests.put("#x { text-shadow: white 2px 5px; }", "#x { text-shadow: white 2px 5px; }");
 		propertyTests.put("#x { text-shadow: 5px 10px; }", "#x { text-shadow: 5px 10px; }");
 		propertyTests.put("#x { text-shadow: 1px 1px 2px 1px black; }", "#x { }");
+		// not possible to parse a comma separated list?
+		//propertyTests.put("#x { text-shadow: 1px 1px 2px red, 0 0 1em blue, 0 0 0.2em blue; }", "#x { text-shadow: 1px 1px 2px red, 0 0 1em blue, 0 0 0.2em blue; }");
```

Comment:
might just not be implemented yet.

Reply:
Yeah, it's still not supported yet, but I don't like texts glowing like rainbow anyway, will leave it to future.